src/autodev.ts

```diff
@@ -102,6 +102,16 @@ const autoDev = async (): Promise<void> => {
   const commitDate = await execAndSlurp(
     `git show -s --format='%ci' origin/${base}`
   )
+  const existingBranch = await execAndSlurp(
+    `git branch -l ${branch} --format='%(refname:short)'`
```

Comment:
Did you test that command? For my local git command, the result in repos is empty until I check out this branch. So I suspect that we need to ensure that remote tracking branches are also included?